src/TaskViewSection/CategoriesMenu/CategoriesMenu.tsx

```diff
@@ -0,0 +1,22 @@
+import { v4 as uuid } from 'uuid'
+import CategoryTab from './CategoryTab'
+
+import { useTaskContext } from '../../Contexts/TaskContext'
+
+const CategoriesMenu = (): JSX.Element => {
+    const { categoryTabs } = useTaskContext()
```

Comment:
let's discuss on the call if we need all of these contexts, I'd say next step is to simplify component hierarchy and remove some of them